src/v1.ts

```diff
@@ -14,7 +15,22 @@ export function NewClient(
   security: ClientSecurity = ClientSecurity.SECURE
 ) {
   const creds = util.createClientCreds(endpoint, token, security);
+  return NewClientWithChannelCredentials(endpoint, creds);
+}
+
+export function NewClientWithCustomCert(
```

Comment:
Please add doc comments to this function and the one below

Reply:
`NewClient` doesn't have any comments. Do you have an example of what you want?

Reply:
```
/**
 * NewClientWithCustomCert creates a new client for calling Authzed APIs using a custom TLS certificate
 */
 ```

Reply:
Added.